lib/actions/jlinkTargetActions.js

```diff
@@ -278,14 +278,14 @@ function writeHex(serialNumber, hexString) {
 
 // Calls nrfprog.recover().
 export function recover() {
-    return (dispatch, getState) => new Promise(resolve => {
+    return (dispatch, getState) => new Promise((resolve, reject) => {
         const appState = getState().app;
         const serialNumber = parseInt(appState.target.serialNumber, 10);
 
         dispatch(targetActions.writeProgressStartAction());
         if (!serialNumber) {
             logger.error('Select a device before recovering');
-            return;
+            reject();
```

Comment:
Change `reject()` to `return reject()`, because calling `nrfjprog.recover()` is not prevented otherwise. Same thing at the next `reject()`.

Reply:
https://stackoverflow.com/questions/32536049/do-i-need-to-return-after-early-resolve-reject
I think I should write like
```
reject();
return;
```
right?
Otherwise, eslint throws warning.